redis/lock.py

```diff
@@ -224,7 +224,7 @@ def owned(self):
         # need to always compare bytes to bytes
         # TODO: this can be simplified when the context manager is finished
         if stored_token and not isinstance(stored_token, bytes):
-            encoder = self.redis.connection_pool.get_encoder()
+            encoder = self.redis.get_encoder()
```

Comment:
I'm beginning to confuse myself a bit -- I think this is a safe change since both redis classes have the `.get_encoder()` method defined on them already? But I'm not positive

Reply:
I think you're right. Since merging in cluster and asyncio, we actually need to start cleaning and unifying these interfaces. There's a lot of duplication - but the philosophy was first get it in, then clean it up.
@dvora-h concur?